src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -35,7 +37,12 @@ internal static partial class CFamilyHelper
         public const string CPP_LANGUAGE_KEY = "cpp";
         public const string C_LANGUAGE_KEY = "c";
 
-        internal static readonly string PchFilePath = Path.Combine(Path.GetTempPath(), "SonarLintForVisualStudio.PCH.preamble");
+        internal static string WorkingDirectory { get; } = Path.GetTempPath();
+        internal static readonly string PchFilePath = Path.Combine(WorkingDirectory, "SonarLintForVisualStudio.PCH.preamble");
+        internal static readonly string RequestConfigFilePath = Path.Combine(WorkingDirectory, "sonar-cfamily.request.config");
+        internal static readonly string ReproducerFilePath = Path.Combine(WorkingDirectory, "sonar-cfamily.reproducer");
```

Comment:
And the two file paths probably shouldn't be readonly.

Reply:
Why not? 🤔 I don't think it's likely for the temp file to change mid VS-session (or at all)